tests/nn/data_parallel/test_sharded_ddp_features.py

```diff
@@ -408,9 +408,9 @@ def closure():
 
 @skip_if_no_cuda
 @skip_if_single_gpu
-def test_gpt2():
-    # Check that the ShardedDDP wrapper accepts tuple(tensors) as inputs
-    world_size = 2
+@pytest.mark.parametrize("world_size", [1, 2])
```

Comment:
Won't this test be skipped for single GPU? I am guessing you wanted to test world_size=1?

Reply:
hmm it does run world_sizes 1 and 2 on a machine having 2+ gpus, not worse than before right ? it does test what the PR changed